.travis.yml

```diff
@@ -3,4 +3,3 @@ node_js:
   - '12'
   - '10'
   - '8'
-  - '6'
```

Comment:
This is unrelated to this PR and should be done afterwards. Should also be done in package.json.